src/views/EventInitiation.vue

```diff
@@ -0,0 +1,231 @@
+<template>
+  <div class="page-container">
```

Comment:
Somehow the page is scrollable vertically - I dont think we want that~~

Reply:
on mobile?

Reply:
Looks like on either views, just dependent on view height